src/view/multipagesview.cpp

```diff
@@ -439,12 +439,12 @@ void MultiPagesView::wheelEvent(QWheelEvent *e)
         return;
 
     int page = m_pageIndex;
-    if (e->delta() > 0) {
-        if (page - 1 >= 0)
-            --page;
-    } else if (e->delta() < 0) {
-        if (page + 1 < m_pageCount)
-            ++page;
+    const QPoint angleDelta = e->angleDelta();
+    int scrollLen = qAbs(angleDelta.x()) > qAbs(angleDelta.y()) ? angleDelta.x() : angleDelta.y();
```

Comment:
```suggestion
    int scrollLen = (qAbs(angleDelta.x()) > qAbs(angleDelta.y()) ? angleDelta.x() : angleDelta.y()) / 8;
```

> Most mouse types work in steps of 15 degrees, in which case the delta value is a multiple of 120; i.e., 120 units * 1/8 = 15 degrees. [ref](https://doc.qt.io/qt-6/qwheelevent.html#angleDelta)

使用原始的 angleDelta 可能会导致过于灵敏，建议考虑 / 8，对于灵敏的设备，小于这个阈值即不会触发滚动。

Reply:
好了